iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/SubstationXml.java

```diff
@@ -73,7 +79,10 @@ protected SubstationAdder createAdder(Network network) {
 
     @Override
     protected Substation readRootElementAttributes(SubstationAdder adder, NetworkXmlReaderContext context) {
-        Country country = context.getAnonymizer().deanonymizeCountry(Country.valueOf(context.getReader().getAttributeValue(null, "country")));
+        Country country = null;
+        if (context.getReader().getAttributeValue(null, COUNTRY) != null) {
```

Comment:
context.getReader().getAttributeValue(null, COUNTRY) could be done only one time